packages/web-features-figuring/src/browser-compat-data/browser.test.ts

```diff
@@ -0,0 +1,38 @@
+import assert from "node:assert/strict";
+
+import { browser } from "./browser";
```

Comment:
This object will change as we upgrade BCD. In mdn-bcd-collector we use mock BCD data for tests. That does have downsides however, it means the mock data could get out of sync with the BCD schema.

A middle ground could be if we had a way to dump the data we're testing so that we would notice if it changed, and can review the changes. Most of the time it won't invalidate the tests.

https://jestjs.io/docs/snapshot-testing is the style of testing I have in mind, but I don't know how to to it with Mocha or get tooling support for updating such tests.

A more basic approach is to prepare a BCD object from the real BCD by deleting all the parts we don't want, and then having a single `assert.deepStrictEqual()` test where we show what the resulting BCD tree looks like. The messy part is updating that test, it would involve logging the tree in the failing test and the pasting it into the code. That's the part that proper tooling support handles.

Thoughts?